assets/chezmoi.io/docs/reference/command-line-flags/global.md

```diff
@@ -40,14 +40,18 @@ changes that would be made without making them.
 
 Make changes without prompting.
 
-## `-h`, `--help`
+## `--interactive`
 
-Print help.
+Prompt before applying each target.
 
 ## `-k`, `--keep-going`
 
 Keep going as far as possible after a encountering an error.
 
+## `--mode` `file`|`symlink`
```

Comment:
This should probably be in `common.md`, not in `global.md`.

Reply:
Same.